exchanges/deribit/deribit.go

```diff
@@ -2393,6 +2393,9 @@ func (d *Deribit) ExecuteBlockTrade(ctx context.Context, timestampMS time.Time,
 	if len(trades) == 0 {
 		return nil, errNoArgumentPassed
 	}
+	if timestampMS.IsZero() {
```

Comment:
VerifiyBlockTrade is called below so there are a double up on these checks, can probably remove them and also target coverage for trade data.